hygen/templates/cy/new/Test.ejs.t

```diff
@@ -61,11 +61,14 @@ describe('in <%= Name %>', () => {
   ///////////////////////////////
   it('edit a <%= Name %>', () => {
 
-    cy.usePactIntercept({
-      method: 'GET',
-      url: '**/isbc/api/<%= name %>s/1',
-      response: {...<%= name %>Item},
-    }).as('get<%= Name %>-1');
+    cy.usePactIntercept(
+      {
+        method: 'GET',
+        url: '**/isbc/api/<%= name %>s/1',
```

Comment:
it's not always isbc

Reply:
I know but we've isbc everywhere. This will be changed on it's own task so that it receives the value as a command line argument